src/domains/ecommerce-app/api/ecommerce-payment-methods-service/v2/_openapi.json.tpl

```diff
@@ -0,0 +1,373 @@
+    "description": "This microservice handles payment methods.",
+    "contact": {
+      "name": "pagoPA - Touchpoints team"
+    }
+  },
+  "servers": [
+    {
+      "url": "https://${hostname}"
+    }
+  ],
+  "tags": [
+    {
+      "name": "payment-methods",
+      "description": "Api's for handle payment methods",
+      "externalDocs": {
+        "url": "https://pagopa.atlassian.net/wiki/spaces/I/pages/611516433/-servizio+payment+methods+service",
+        "description": "Technical specifications"
+      }
+    }
+  ],
+  "externalDocs": {
+    "url": "https://pagopa.atlassian.net/wiki/spaces/I/pages/492339720/pagoPA+eCommerce+Design+Review",
+    "description": "Design review"
+  },
+  "security": [
+    {
+      "ApiKeyAuth": []
+    }
+  ],
+  "paths": {
+    "/payment-methods/{id}/fees": {
+      "post": {
+        "tags": [
+          "payment-methods"
+        ],
+        "operationId": "calculateFees",
+        "summary": "Calculate payment method fees",
+        "description": "GET with body payload - no resources created: Return the fees for the choosen payment method based on transaction amount etc.\n",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "Payment Method ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "maxOccurrences",
+            "in": "query",
+            "description": "max occurrences",
+            "required": false,
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "x-transaction-id",
+            "in": "header",
+            "schema": {
+              "type": "string"
+            },
+            "required": true,
+            "description": "The ecommerce transaction id"
+          }
+        ],
+        "requestBody": {
+          "$ref": "#/components/requestBodies/PostPaymentMethodPSP"
+        },
+        "responses": {
+          "200": {
+            "description": "Return list of psp ordered by fee.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/CalculateFeeResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Resource not found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
+          },
+          "500": {
```

Comment:
add 504 error code since those are api exposed through APIM and this can be returned when a gateway timeout has occurred